client/blocks/promo-card-block/index.jsx

```diff
@@ -40,7 +35,7 @@ const PromoCardBlock = ( {
 	return (
 		<>
 			<QueryJetpackPlugins siteIds={ [ selectedSiteId ] } />
-			{ jetpackNonAtomic || selectedPlugin || isFetching ? (
```

Comment:
We removed the business logic from the component as /status page is already doing it. [The only other place](https://github.com/Automattic/wp-calypso/blob/add/stats-blaze-banner/client/my-sites/marketing/traffic/index.js#L53) which is reusing this component[ will have Blaze banner](https://github.com/Automattic/wp-calypso/pull/71276) on it, so this logic can be safely removed from here.